### DISCUSSION TITLE
Welcome to MindsDB Discussions!

### DISCUSSION THREAD

**Discussion by ZoranPandovski** [mindsdb/mindsdb#1219] [General]
## 👋 Welcome to MindsDB community discussions :bear:
This is a place to ask questions, get help, share ideas, provide feedback and just talk.
  * If you found a bug, it is best to submit an issue to ensure it is resolved quickly.

As a reminder, the [code of conduct](https://github.com/mindsdb/mindsdb/blob/stable/CODE_OF_CONDUCT.md) still applies to GitHub Discussions. Let’s make it a place that everyone feels welcome :raised_hands: !

**Comment by sotolou2022** [comment]
I installed it on docker, not seeing much as far as tutorials etc..
